### PR TITLE
Documentation note for large-message-frame-size

### PR DESCRIPTION
**Pull request by toaditoad**
Added to docs that large-message-frame-size is restricted to 1/8th of aeron.term.buffer.length #22257.

### CODE REVIEW

File: akka-remote/src/main/resources/reference.conf [hunk 1]
```diff
@@ -783,6 +783,8 @@ akka {
         buffer-pool-size = 128
 
         # Maximum serialized message size for the large messages, including header data.
+        # It is currently restricted to 1/8th the size of a term buffer what can be
```
Comment (sebastianharko) [inline akka-remote/src/main/resources/reference.conf]: did you mean "that can be configured" instead of "what can be configured" ?

Comment (toaditoad) [inline akka-remote/src/main/resources/reference.conf]: @sebastianharko, thanks for pointing that out! My bad...

**Review by 2m — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**